app/services/integration/kubernetes_service.rb

```diff
@@ -124,7 +124,7 @@ def labels_for_provider(entry)
   end
 
   class Route < K8s::Resource
-    def initialize(attributes, **options)
+    def initialize(attributes, options = {})
```

Comment:
I don't like this hack... Some info on why it is here is in the PR description.

Maybe we need to switch to another k8s client library :shrug:

Reply:
I think the point was that gem is outdated and a potential maintenance issue going forward.